### ISSUE TITLE
Not compatible with Firefox Quantum

### ISSUE THREAD

**Issue by magjac** [mbostock/bl.ocks.org#12]
When visiting https://addons.mozilla.org/en-US/firefox/addon/blocksorg/, with Firefox Quantum 57.0 (64-bit) on Windows 7, I get the message "Not compatible with Firefox Quantum" and cannot add the add-on to Firefox.

**Comment by mbostock** [comment]
Yeah, I’m not totally surprised. Not sure when I’ll get around to fixing the add-on, however.

**Comment by erikvangameren** [comment]
Hope you get it fixed! Would love to use it

**Comment by jfreedman0** [comment]
~~Created a bare bones web extension port that works in Firefox Quantum. #13~~ I closed my PR because the chrome extension is actually written as a web extension, so simply using the chrome extension will work on Firefox Quantum.

**Comment by jfreedman0** [comment]
@mbostock Simply needs to have it signed and submit it to Firefox. In the meantime, you can clone the repo, cd to the chrome extension directory that contains manifest.json and run `web-ext build`. Then you can add the extension manually in Firefox Developer Edition. The regular Firefox will not allow it to be added because it is not signed.

**Comment by jfreedman0** [comment]
Uploaded a V2 to Firefox and included credit to @mbostock  in the license: https://addons.mozilla.org/en-US/firefox/addon/bl-ocks-org/
